### ISSUE TITLE
Broken VMAlert deployment if `VMAlert.spec.extraArgs."notifier.config"` is set

### ISSUE THREAD

**Issue by tamcore** [VictoriaMetrics/operator#990]
Since https://github.com/VictoriaMetrics/VictoriaMetrics/issues/2690 we deploy our VMAlert with
```yaml
apiVersion: operator.victoriametrics.com/v1beta1
kind: VMAlert
spec:
  ....
  extraArgs:
    notifier.config: /etc/vm/secrets/victoria-metrics-agent-vmalert-notifiers/notifier.yaml
```
and during our upgrade attempt from v1.93.15 LTS to v1.101.0 VMAlert failed on us with
`failed to init: failed to init notifier: only one of -notifier.config or -notifier.url flags must be specified`, since even though we don't define `notifier.url`, the operator still enforces the parameter.

From the resulting deployment:
```yaml
        - -httpListenAddr=:8080
        - -notifier.config=/etc/vm/secrets/victoria-metrics-agent-vmalert-notifiers/notifier.yaml
        - -notifier.url=
        - -remoteRead.basicAuth.passwordFile=/etc/vmalert/remote_secrets/REMOTEREAD_BASICAUTHPASSWORD
```

Removing `-notifier.url=` from the container's args, seems to work just fine.

**Comment by Haleygo** [comment]
Hi,
yes, `-notifier.config` should be set in `.spec.notifierConfigRef`, adding extra check in admission webhook [here](https://github.com/VictoriaMetrics/operator/pull/993).

**Comment by Haleygo** [comment]
The extra check is included in [v0.46.1](https://github.com/VictoriaMetrics/operator/releases/tag/v0.46.1), close as completed. 
Feel free to reopen if there are further questions.
